project/src/main/java/com/google/sps/utilities/WrapperDatabaseService.java

```diff
@@ -0,0 +1,30 @@
+package com.google.sps.utilities;
+
+import com.google.cloud.spanner.DatabaseClient;
+import com.google.cloud.spanner.DatabaseId;
+import com.google.cloud.spanner.Spanner;
+import com.google.cloud.spanner.SpannerOptions;
+
+public class WrapperDatabaseService implements DatabaseService {
+  public static final String INSTANCE_ID = "step-188-instance";
```

Comment:
Use the DatabaseConstants class that's on master rn

Reply:
+1

Reply:
Done - thanks.